docs/en/modules/net_info.md

```diff
@@ -0,0 +1,79 @@
+# net_info Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-01-21 | [smcl](http://blog.mclemon.io/esp8266-contributing-to-the-nodemcu-ecosystem) | [wolfgangr](https://github.com/wolfgangr/nodemcu-firmware) | [net_info.c](../../../app/modules/net_info.c)
```

Comment:
By contributing a new module you automatically become its 1st maintainer. With close to 50 modules we just don't have the capacity to maintain someone else's code w/o the commitment from the contributor.